### DISCUSSION TITLE
Error when compiling generator

### DISCUSSION THREAD

**Discussion by DenHayes** [prometheus/snmp_exporter#1158] [Q&A]
I'm getting the following error when trying to compile generator and wonder if anyone has run into this before. 

dennis@dev:~/snmp_exporter/generator$ make generator
go build
../config/config.go:24:2: //go:build comment without // +build comment
make: *** [Makefile:87: generator] Error 1

This is being done on Ubuntu 22.04.4 LTS

Any suggestions on how to get this to build?

**Comment by SuperQ** [comment]
That line does not exist in the [source code](https://github.com/prometheus/snmp_exporter/blob/01f11ea675349407425d7b61521d5058cb1961b0/config/config.go#L24). Are you sure you haven't made local modifications?

**Reply by DenHayes** [reply]
Nope, no modifications.  I even deleted everything and started from scratch; re-cloning snmp-exporter from github.

**Reply by DenHayes** [reply]
I tried on a different server and it worked perfectly.  Both are Ubuntu 22.04 LTS; there must be a missing module on the one server.
